### ISSUE TITLE
403 on Clone

### ISSUE THREAD

**Issue by diego-ortega-leroymerlin** [desktop/desktop#9055]
### Describe the bug

Failing to clone with a 403 response.

### Version & OS

Tryin with gitBash and the Desktop version 2.3.1. with no success. Windows 10 Pro.

### Steps to reproduce the behavior

The repository is from an organization from which I'm subscribed, I have tryed to clone it opening the desktop app from the web or just opening it with a standard clone, but the opening part and sending request is no problem. The problem is the authentification.
I keep trying with gitBash and the Desktop version with no success.
I have tryed with username, email and changing password, but didnt work.
I can access in the web to the repository and in the desktop version it seems like I'm logged in, but when I try to clone It asks the user and pass and when i give them it returns 403.

### Screenshots

In github desktop:
![image](https://user-images.githubusercontent.com/60612281/73829266-fa9dc700-4802-11ea-99aa-6f65fda1f10d.png)

### Logs

In gitbash:
$ git clone https://github.com/adeo/TOM.git
Cloning into 'TOM'...
remote: The `adeo' organization has enabled or enforced SAML SSO. To access
remote: this repository, visit https://github.com/orgs/adeo/sso?authorization_request=AOON5OK3ZUKKNIBCF7ACZTK6HKIXZA5PN5ZGOYLONF5GC5DJN5XF62LEZYAHFM73VVRXEZLEMVXHI2LBNRPWSZGOC2WRSJ5PMNZGKZDFNZ2GSYLML52HS4DFVNHWC5LUNBAWGY3FONZQ
remote: and try your request again.
fatal: unable to access 'https://github.com/adeo/TOM.git/': The requested URL returned error: 403

**Comment by diego-ortega-leroymerlin** [comment]
The auth of the enforced SAML SSO wasn't working well, had to access with the link showed just after the failed request on the gitbash and retry on gitbash. On the desktop version doesn't work.

**Comment by steveward** [comment]
@diego-ortega-leroymerlin did you attempt to log in to your account via `File` > `Options` > `Accounts` in GitHub Desktop? You won't be able to authenticate via the `Authentication failed` dialog.

**Comment by diego-ortega-leroymerlin** [comment]
Yes I did, it doesn't have anything to do with being logged in or not as i
said.
*The problem was that*, even while I was on the organization group and I
could get to the repositories, for some reason *I didn't get the correct
auth token in the organization's validation process* (I did it several
times to no avail), so when I wanted to clone I was refused.
Only *after doing it on the gitbash and opening the indicated link did it*
kick again, this time getting the token (even when what you see doesn't
change).
